config/doctrine/File.orm.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
What if we use a symlink here?

Reply:
Unfortunately, Doctrine uses a `is_file` call here: https://github.com/doctrine/persistence/blob/2.2.x/lib/Doctrine/Persistence/Mapping/Driver/SymfonyFileLocator.php#L151 That means that a symlink will work in neither location, as `is_file` returns false for symlinks.